eng/pipelines/templates/jobs/code.gen.yml

```diff
@@ -0,0 +1,71 @@
+parameters:
+- name: SDKType
+  type: string
+  default: all
+- name: TestSetupSteps
+  type: stepList
+  default: []
+- name: TestTimeoutInMinutes
+  type: number
+- name: Matrix
+  type: string
+- name: DependsOn
+  type: string
+  default: ''
+- name: UsePlatformContainer
+  type: boolean
+  default: false
+- name: CloudConfig
+  type: object
+  default: {}
+- name: ProjectListOverrideFilePropertyName
+  type: string
+  default: ''
+- name: ServiceDirectory
+  type: string
+  default: ''
+
+jobs:
+  - job:
+    displayName: "Code Generation"
```

Comment:
Based on our conversation we might not want to limit this to just Code Generation. We might want to make it a little more generic.